tests/functional/floorplan_reconstruction/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+"""Tests TAPA floorplan reconstruction."""
+
+# Copyright (c) 2025 RapidStream Design Automation, Inc. and contributors.
+# All rights reserved. The contributor(s) of this file has/have agreed to the
+# RapidStream Contributor License Agreement.
+
+load("//bazel:pytest_rules.bzl", "py_test")
+
+py_test(
+    name = "vadd-get-slot-test",
+    srcs = ["get-slot-test.py"],
+    args = ["--test=vadd"],
+    data = [
+        "conftest.py",
+        "golden/vadd-golden.json",
+        "graph/vadd-graph.json",
+        "slot/vadd-slot.json",
+    ],
+    deps = [
+        "//tapa",
```

Comment:
Don't you only need `//tapa/common`?

Minimizing deps helps improve bazel's cache hit rate.